### PR TITLE
Create BookReview

### PR DESCRIPTION
**Pull request by clerav**
Will need to implement absolute path to detail view of BookReview similar to club model

### CODE REVIEW

File: BookClub/admin.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from BookClub.models.club import Club
 from BookClub.models.club_membership import ClubMembership
 from BookClub.models.book import Book
+from BookClub.models.review import BookReview
 from BookClub.models.booklist import BookList
 from BookClub.models.meeting import Meeting
```
Comment (pleofthegem) [inline BookClub/admin.py:9]: Maybe change to just from BookClub.models import *
instead of having to import each model separately

**Review by pleofthegem — DISMISSED** [review]
imports for admin

Everything else looks great

**Review by amir-rahim — APPROVED** [review]
👌 done!

**Review by Moose3245 — APPROVED** [review]
👍

**Pull request merged**